samples/nrf9160/at_client/README.rst

```diff
@@ -37,24 +37,22 @@ Building and running
 Connecting to the nRF9160 DK board with LTE Link Monitor
 ========================================================
 
-To connect to the nRF9160 DK board with LTE Link Monitor, perform the following steps:
-
-#. Launch the LTE Link Monitor after programming the AT Client sample onto the nRF9160 DK board.
-#. Make sure that **Automatic requests** is enabled in LTE Link Monitor.
-#. Connect the nRF9160 DK board to the PC with a USB cable.
-#. Power on the nRF9160 DK board.
-#. Click **Select Device** and select the particular board device entry from the drop-down list in the LTE Link Monitor.
-#. Observe that initially the command :command:`AT+CFUN?` is automatically sent to the modem, which returns a value 4, indicating that the modem is in the offline mode.
-#. Observe that the LTE Link Monitor terminal display also shows :command:`AT+CFUN=1` indicating that the modem has changed to the normal mode.
-#. Press the reset button on the nRF9160 DK to reboot the board and start the AT Client sample.
-
+See :ref:`lte_connect` for connecting to the nRF9160 DK board with LTE Link Monitor.
 
 Testing
 =======
 
 After programming the sample to your board, test the sample by performing the following steps:
 
 1. `Connect to the nRF9160 DK board with LTE Link Monitor <Connecting to the nRF9160 DK board with LTE Link Monitor_>`_.
+
+   .. note::
+
+      Make sure that **Automatic requests** is enabled in LTE Link Monitor.
+
+#. Initially the command AT+CFUN? is automatically sent to the modem, which returns a value 4, indicating that the modem is in the offline mode.
+#. The LTE Link Monitor terminal display also shows AT+CFUN=1 indicating that the modem has changed to the normal mode.
```

Comment:
In reality the line `AT+CFUN=1` is a command sent from the Link Monitor (automatically or by the user) to the modem, to which `OK` is the reply. In case `AT+CFUN?` query command is sent then the modem replies by `+CFUN: 1` to indicate current mode.

Something like this would be closer to what happens:

The LTE Link Monitor terminal display also shows `AT+CFUN=1` followed by `OK` indicating that the modem was changed to the normal mode.

Reply:
Changed the sentence according to the suggestion.